drizzlepac/haputils/deconvolve_utils.py

```diff
@@ -479,7 +479,7 @@ def _create_input_psf(psf_name, calimg, total_flux):
     lib_size = [lib_psf_arr.shape[0] // 2, lib_psf_arr.shape[1] // 2]
 
     # create hamming 2d filter to avoid edge effects
-    h = ss.hamming(lib_psf_arr.shape[0])
+    h = windows.hamming(lib_psf_arr.shape[0])
```

Comment:
is this backward compatible?

Reply:
Yes.

Reply:
@mcara It is just an import and namespace change.  Please see PR#1848.

Reply:
Why not use the similar function from `numpy`?

Reply:
https://numpy.org/doc/stable/reference/generated/numpy.hamming.html

Reply:
@mcara I could consider this for the future!  However, we are in the middle of a build sequence, so I am not inclined to change anything too dramatically at the last minute.